nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/cluster/nf-cluster-table.js

```diff
@@ -853,6 +852,14 @@
                 var aNode = formatNodeAddress(a);
                 var bNode = formatNodeAddress(b);
                 return aNode === bNode ? 0 : aNode > bNode ? 1 : -1;
+            } else if (sortDetails.columnId === 'version' || sortDetails.columnId === 'javaVersion') {
```

Comment:
The field `id` is `javaversion` all lowercase, versus the `field` value, which is `javaVersion`, so it looks like this needs to be changed.

Reply:
Thanks for reviewing! If you look at line 1454, the `columnId` is set to the `field` vs the `id`. All of the columns in line 835, which has not been changed, also use the `field` value.